docker-for-mac/index.md

```diff
@@ -16,12 +16,11 @@ title: Get started with Docker Desktop for Mac
 
 Welcome to Docker Desktop!
 
-This page contains information about Docker Desktop Community (Stable and Edge) releases. For information about Docker Desktop Enterprise (DDE) releases, see [Docker Desktop Enterprise](/ee/desktop/).
+The _Docker Desktop for Mac_ section contains information about the Docker Desktop Community Stable release. For information about Edge releases, see [Edge release notes](edge-release-notes/). For information about Docker Desktop Enterprise (DDE) releases, see [Docker Desktop Enterprise](/ee/desktop/).
```

Comment:
Maybe something more like 'for information on the differences in the Edge version please see the Edge release notes'

Reply:
Sure. How about "For information about the features available in Edge releases, see the Edge release notes."?